src/content/docs/style-guide/writing-strategies/install-docs-guidance.mdx

```diff
@@ -1,5 +1,5 @@
 ---
-title: How to document install methods
+title: How to document our install methods
```

Comment:
Based on this title, I thought the doc would include something more action-oriented for tech writers rather than an information overview for install methods. 

I'm wondering if we can add another ticket (or continue to work on this doc) that clarifies some ambiguous action items here, e.g.:

- "For the docs, we should send people to guided install whenever it's appropriate to do so. That is, when guided install is likely to lead to a successful outcome."

How can a writer who has never documented install methods make this call? Maybe we can add something about always reaching out to engineers to clarify why the guided install is structured the way it is. Something else that COULD be helpful (not sure if we want it documented) is maybe some context on guided install being a work in progress (WIP) so writers know why exactly we should cross reference with SMEs for this especially.

- "They also provide the most straightforward installation path for standard use cases.

This method is less effective for customers who have more complicated or obscure things they're trying to monitor or non-standard systems."

Same comment as above. Might be worth getting to the bottom of the WHO is guided install best for -- I've heard mixed stories between different engineering teams + our tech writer team. Also, "straightforward install path" almost contradicts the "most successful" qualifier in the section above. If manual install is the most straight forward, then why bother with guided install (from the POV of a writer checking out this page). Could be worth teasing this apart, or if we don't have those answers right now, reiterating who our best resources are to get the information we need.

- "That being said, the current state of the manual installation docs is inconsistent and, in some cases, like the APM agent docs, bewilderingly complicated." 

Part of me wants to push against this but I'm not sure why/how LOL. Maybe we can come up with a more diplomatic way to say these are works in progress, or we're hoping to phase them out, or they're mostly there if the other two fail. ("Method of last resort" kind of drives at this, but could be helpful to say it more outright. AFAIK, the docs install is good if you're having troubleshooting errors and you want more hands-on to the individual files to figure out what's failing the install? Idk)

Reply:
Didn't realize how long this comment is. Just some food for thought!

Reply:
This is a great comment, @akristen ! A lot of good food for thought here. I have a ticket to try to uncover some of the answers to these questions. I'll drop a link to your comment in that ticket.

Unfortunately, I don't know some of the answers to your great questions!

Reply:
I think you're right thought about being more diplomatic. :D I'll rewrite that sentence.